Packet++/header/IPLayer.h

```diff
@@ -30,5 +30,8 @@ namespace pcpp
 
 		/// An empty destructor
 		virtual ~IPLayer() = default;
+
+		/// A static method that gets a pointer to the IP data header and returns the IP version
```

Comment:
Since it's a public method (although used internally), let's add the params and the return type to the doxygen documentation?
```cpp
/// A static method that gets a pointer to the IP data header and returns the IP version
/// @param[in] data ...
/// @param[in] dataLen ...
/// @return ...
```